public/javascripts/i18n.js

```
@@ -0,0 +1,1080 @@
+    var lazyEvaluate = function (message, scope) {
+        if (isFunction(message)) {
+            return message(scope);
+        }
+        return message;
+    };
+
+    var merge = function (dest, obj) {
+        var key; var
+            value;
+        for (key in obj) {
+            if (obj.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)